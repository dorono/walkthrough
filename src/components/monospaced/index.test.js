import React from 'react';
import {shallow} from 'enzyme';
import Monospaced from './index';

describe('Monospaced', () => {
    it('should render without errors', () => {
        shallow(<Monospaced />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Monospaced />)).toMatchSnapshot();
    });
});
