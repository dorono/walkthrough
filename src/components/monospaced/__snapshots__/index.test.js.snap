// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Monospaced should match snapshot 1`] = `
<span
  className="root"
/>
`;
